java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -183,9 +183,10 @@ public String toString() {
    * @param filePath          the path of the file to read, or null if no path should be read.
    * @param address           the address of the buffer to read from or 0 if we should not.
    * @param length            the length of the buffer to read from.
+   * @param timeUnit              return type of TimeStamp in units
```

Comment:
nit, spacing a bit off here.

Reply:
@abellina Thanks for noticing this :) PTAL